tsfc/coffee.py

```diff
@@ -16,41 +16,44 @@
 
 from gem import gem, impero as imp
 
-from tsfc.constants import SCALAR_TYPE, PRECISION
+from tsfc.constants import SCALAR_TYPE
 from tsfc.logging import logger
 
 
 class Bunch(object):
     pass
 
 
-def generate(impero_c, index_names, roots=(), argument_indices=()):
+def generate(impero_c, index_names, parameters, roots=(), argument_indices=()):
     """Generates COFFEE code.
 
     :arg impero_c: ImperoC tuple with Impero AST and other data
     :arg index_names: pre-assigned index names
+    :arg parameters: TSFC parameters
     :arg roots: list of expression DAG roots for attaching
         #pragma coffee expression
     :arg argument_indices: argument indices for attaching
         #pragma coffee linear loop
         to the argument loops
     :returns: COFFEE function body
     """
-    parameters = Bunch()
-    parameters.declare = impero_c.declare
-    parameters.indices = impero_c.indices
-    parameters.roots = roots
-    parameters.argument_indices = argument_indices
+    params = Bunch()
+    params.declare = impero_c.declare
+    params.indices = impero_c.indices
+    params.roots = roots
+    params.argument_indices = argument_indices
```

Comment:
Seemed to me a bit more consistent with some other piece of code: `parameters` for passed tsfc parameters and `params` for module internals...